README.md

```diff
@@ -1,16 +1,16 @@
 # Android Food Scanner App 🍌
 
-*The native, Android [Food Scanner App](https://play.google.com/store/apps/details?id=com.lifyzer) 🥝*
+<div align="center" markdown="1">The native, Android [Food Scanner App](https://play.google.com/store/apps/details?id=com.lifyzer) 🥝*
 
-![Healthy Food, Healthy Tomato](extras/assets/broccoli-healthy-food.svg)
+![Healthy Food, Healthy Tomato](extras/assets/broccoli-healthy-food.svg)</div>
 
 
 ## About The App 🤔
 
-* Lifyzer App uses OCR and barcode scanner technologies.
+<div align="center" markdown="1">Lifyzer App uses OCR and barcode scanner technologies.
```

Comment:
We could use `style="align:center` which is the only HTML5's valid way

Reply:
Feel free to build upon. I didn't want to introduce too much non-Markdown, so I used the old-fashioned approach (OK, call it "quick-and-dirty" :speak_no_evil:)

Reply:
No worries :)